cmd/expiration-mailer/main.go

```diff
@@ -447,15 +447,16 @@ func initStats(stats prometheus.Registerer) mailerStats {
 	sendLatency := prometheus.NewHistogram(
 		prometheus.HistogramOpts{
 			Name:    "send_latency",
-			Help:    "Time the mailer takes sending messages",
+			Help:    "Time the mailer takes sending messages in seconds",
 			Buckets: metrics.InternetFacingBuckets,
 		})
 	stats.MustRegister(sendLatency)
 
 	processingLatency := prometheus.NewHistogram(
 		prometheus.HistogramOpts{
-			Name: "processing_latency",
-			Help: "Time the mailer takes processing certificates",
+			Name:    "processing_latency",
+			Help:    "Time the mailer takes processing certificates in seconds",
+			Buckets: []float64{0.01, 0.2, 0.5, 1, 2, 5, 10, 20, 60, 75, 90, 120},
```

Comment:
We recently cut down on some histogram buckets because of the increased cardinality. I think if we are measuring the difference between 0.01 seconds and 120 seconds we don't need some of the ones in between. If you don't care about things much more than 45 seconds you could use the bucket definition we use for other internet facing services `Buckets: metrics.InternetFacingBuckets`.

Reply:
I definitely want to have the metrics >=60 seconds.